mtp_cashbook/templates/cashbook/dashboard.html

```diff
@@ -10,7 +10,7 @@
         <h2>{% trans "New" %}</h2>
         <p>
           {% blocktrans with payments_count="<strong>{0}</strong>"|safewrap:new_transactions %}
-          {{ payments_count }} Payments to credit
+          {{ payments_count }} Credits to process
```

Comment:
It might be nice if we could do the capitalisation here in CSS (`Credits` is the second word in the sentence), but I can't see a clean way to do that without extra markup?